scene/2d/animated_sprite_2d.cpp

```diff
@@ -261,6 +261,10 @@ void AnimatedSprite2D::set_sprite_frames(const Ref<SpriteFrames> &p_frames) {
 	frames = p_frames;
 	if (frames.is_valid()) {
 		frames->connect("changed", callable_mp(this, &AnimatedSprite2D::_res_changed));
+		Vector<String> animation_names = frames->get_animation_names();
```

Comment:
@akien-mga This returns sorted animation names, as desired (so its first element is indeed what we want):
https://github.com/godotengine/godot/blob/9904a9db5a90f2f1896b6b7e1603ca2dd2745bad/scene/resources/sprite_frames.cpp#L105-L112

But in other places in the source code `SpriteFrames::get_animation_list()` is being used instead and the obtained list is being sorted separately:
https://github.com/godotengine/godot/blob/9904a9db5a90f2f1896b6b7e1603ca2dd2745bad/scene/resources/sprite_frames.cpp#L99-L103
https://github.com/godotengine/godot/blob/9904a9db5a90f2f1896b6b7e1603ca2dd2745bad/editor/plugins/sprite_frames_editor_plugin.cpp#L985-L987

Both approaches seem to result in the same ordering (both end up comparing with `is_str_less`). But I think we should be consistent and use one over the other. I'd guess sorting a list is probably slower (it probably doesn't matter though) but in the list version there's no StringName -> String conversion (which is also cheap I guess). So is any of these approaches preferred?